crates/fuel-core/src/service/genesis/importer/on_chain.rs

```diff
@@ -143,6 +144,8 @@ impl ImportTable for Handler<Transactions> {
         for transaction in &group {
             tx.storage::<Transactions>()
                 .insert(&transaction.key, &transaction.value)?;
+            tx.storage::<ProcessedTransactions>()
+                .insert(&transaction.key, &())?;
```

Comment:
Hmm, this migration even shouldn't exist because we have `OldTransactions` for that. I think we forgot to remove it during https://github.com/FuelLabs/fuel-core/pull/1811.

Hmm, it seems we need to migrate the `ProcessedTransactions` table instead of fetching it from `OldTransactions` because we can't rely on off-chain data during migration. Off-chain data migration is optional and only for nodes that want to support GraphQL API

Reply:
#1811 is not merged yet, so we can still include that there. I'm going to remove this for now, we can do that in a separate PR if not in #1811. Removed in 435d1c66c.